git-multimail/git_multimail.py

```diff
@@ -2373,18 +2439,34 @@ def send_emails(self, mailer, body_filter=None):
                 )
 
 
+def include_ref(refname, ref_filter_regex, is_inclusion_filter):
+  return not (bool(ref_filter_regex.search(refname)) ^ is_inclusion_filter)
+  ## More readable version:
+  #does_match = bool(re.search(ref_filter_regex))
+  #if is_inclusion_filter:
+  #  return does_match
+  #else:  # exclusion filter -- we include the ref if the regex doesn't match
+  #  return not does_match
```

Comment:
Yeah, I tend to prefer 1 line of code over 5, but I suspected this one was pushing the limits.  I'll be happy to switch it.